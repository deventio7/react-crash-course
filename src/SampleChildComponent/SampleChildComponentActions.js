import SampleChildComponentActionTypes from './SampleChildComponentActionTypes';

const updateStoredText = (text) => {
  return {
    type: SampleChildComponentActionTypes.SAMPLE_CHILD_COMPONENT_SUBMIT,
    payload: text
  }
}

export default {
	updateStoredText
}