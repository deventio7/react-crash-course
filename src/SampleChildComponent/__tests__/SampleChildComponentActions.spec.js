import SampleChildComponentActions from '../SampleChildComponentActions';

describe('SampleChildComponentActions', () => {
  it('should dispatch action SAMPLE_CHILD_COMPONENT_SUBMIT', () => {
    const actionPayload = SampleChildComponentActions.submit('TEST VALUE');
    expect(actionPayload).toMatchSnapshot();
  });
});