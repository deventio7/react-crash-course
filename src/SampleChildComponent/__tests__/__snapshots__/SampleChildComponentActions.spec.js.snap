// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleChildComponentActions should dispatch action SAMPLE_CHILD_COMPONENT_SUBMIT 1`] = `
Object {
  "payload": "TEST VALUE",
  "type": "SAMPLE_CHILD_COMPONENT_SUBMIT",
}
`;
